### ISSUE TITLE
Cannot enable apps through UI in 28.0.4

### ISSUE THREAD

**Issue by te-online** [nextcloud/docker#2193]
### Image tag
`nextcloud:28.0.4-fpm-alpine`

### Problem
When trying to activate any app on Nextcloud's apps page, the following error appears:

```
Error: This app cannot be enabled because it makes the server unstable
```

A Content Security Policy error is logged in the console of the browser.

```
Refused to connect to 'http://localhost/apps/files/' because it violates the following Content Security Policy directive: "connect-src 'self'".
```

### Reproduction steps / Setup
- Check-out the repository and change to the `tests` folder. 
https://github.com/te-online/timemanager/tree/main/tests
- Copy the `.env.example` to `.env` and add some values for the variables.
- Change the image tag of the `app` container to the latest
- Run `docker-compose up -d`, followed by `docker exec $(docker ps -qf "name=app") sh -c 'chown www-data:root custom_apps'`.
- Navigate to localhost:8000 and try to activate any apps

---

At this point, I believe this is an issue with the configuration in this image, but it may also just be my proxy configuration that is messing with things. Apologies, if it's user error, but this config has been working for a long time. 

I think it unlikely that it's a core server issue, because then it would likely have been caught in RC, this is why I open the issue here.

Background: I use this as my dev and testing setup and [I can see some people on Reddit](https://www.reddit.com/r/NextCloud/comments/1bqf8wp/2804_error_this_app_cannot_be_enabled_because_it/) having the same issue, though they don't specify if they're using this container.

**Comment by joergmschulz** [comment]
same is true for the 29 beta 4

**Comment by tzerber** [comment]
see https://github.com/nextcloud/server/issues/44685#issuecomment-2083037678 . Works on my end, but keep in mind it's working on my setup, and mine is a bit weird one (two host acting as proxy for a bunch of other stuff).

**Comment by joshtrichards** [comment]
Duplicate of nextcloud/server#44685
Closed by nextcloud/server#44939

**Comment by githubbbie** [comment]
Confirmed again this *IS* working for the latest version of the v28 series looks like: 28.0.9.1

Attaching my docker-compose.yml and hope this helps anyone:

[docker-compose.yml.txt](https://github.com/user-attachments/files/16715070/docker-compose.yml.txt)
